webapp/cypress/e2e/index.cy.js

```
@@ -5,6 +5,13 @@ const selectItem = (selector, itemValue) => {
     .click();
 };
 
+const uploadFile = (selector, inputFiles) => {
+  const files = (Array.isArray(inputFiles) ? inputFiles : [inputFiles]).map(
+    (fileName) => 'webapp/cypress/fixtures/' + fileName
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)